From 254171f360ac0061eb14ca6303b3ce53cdf8f308 Mon Sep 17 00:00:00 2001
From: Carlos Alberto Lopez Perez <clopez@igalia.com>
Date: Mon, 14 May 2018 14:42:03 +0200
Subject: [PATCH] WPEBackend libraries are versioned now.

This fixes the build with WPEBackend after WPEBackend commit
2d53b25913944b6974afb22aeeaed4cc44c665d2
---
 cmake/FindWPE.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/FindWPE.cmake b/cmake/FindWPE.cmake
index c7a93d0..fb050ce 100644
--- a/cmake/FindWPE.cmake
+++ b/cmake/FindWPE.cmake
@@ -29,7 +29,7 @@
 # ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 find_package(PkgConfig)
-pkg_check_modules(PC_WPE QUIET wpe)
+pkg_check_modules(PC_WPE QUIET wpe-0.1)
 
 find_path(WPE_INCLUDE_DIRS
     NAMES wpe/loader.h
@@ -37,7 +37,7 @@ find_path(WPE_INCLUDE_DIRS
 )
 
 find_library(WPE_LIBRARIES
-    NAMES WPEBackend
+    NAMES WPEBackend-0.1
     HINTS ${PC_WPE_LIBDIR} ${PC_WPE_LIBRARY_DIRS}
 )
 
-- 
2.11.0

